server/scripts/rebuild-and-run-standalone-local.sh

```diff
@@ -1,5 +1,69 @@
 #!/bin/bash -e
 
+export DM_DEBUG_COMMANDS=true
+# export DM_DEBUG_KEEP_JOB_FOLDER=true
+
+USE_ANDROID=true
+
+#
+# BEGIN Experimental Android setup
+#
+if [ "$USE_ANDROID" = true ] ; then
+	# https://developer.android.com/tools/variables
+	export ANDROID_ROOT=${HOME}/android
+	export ANDROID_PROGUARD=${ANDROID_ROOT}/proguard.jar
```

Comment:
I guess these variables should be in `server/scripts/standalone/service-standalone.sh` ?